src/ui/MobileSearch/Search.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts" context="module">
+  import { queryProjects } from '@/api/projects'
+  import Asset from './Asset.svelte'
+  import Trend from './Trend.svelte'
+  import Insight from './Insight.svelte'
+  import Person from './Person.svelte'
+  import { FILTER_ITEMS } from './utils'
+  import { queryInsights, queryInsightsBySearchTerm, queryPeople, queryTrends } from './api'
+
+  export const TABS_CONTENT = {
+    [FILTER_ITEMS.Assets]: {
+      query: queryProjects,
+      filter: (searchTerm, items) => {
+        const projects = items.filter(
+          ({ name, ticker }) =>
+            name.toLowerCase().includes(searchTerm) || ticker.toLowerCase().includes(searchTerm),
+        )
+
+        if (!searchTerm) {
+          return items
+        }
+
+        return projects
```

Comment:
`[D-3-0]`
1.`if (!searchTerm) { return items }` is redundant due to `LOC19 of Suggestions.svelte`;
2. return result immediately- `return items.filter(.....`